app/common/data/models/__init__.py

```diff
@@ -190,3 +191,50 @@ class Form(BaseModel):
         UniqueConstraint("title", "section_id", name="uq_form_title_section"),
         UniqueConstraint("slug", "section_id", name="uq_form_slug_section"),
     )
+
+    questions: Mapped[list["Question"]] = relationship(
+        "Question", lazy=True, order_by="Question.order", collection_class=ordering_list("order")
+    )
+
+
+class QuestionDataType(enum.StrEnum):
+    # If adding values here, also update QuestionTypeForm
+    # and manually create a migration to update question_type_enum in the db
+    TEXT_SINGLE_LINE = "A single line of text"
+    INTEGER = "A whole number"
+    TEXT_MULTI_LINE = "Multiple lines of text"
```

Comment:
```suggestion
    TEXT_SINGLE_LINE = "A single line of text"
    TEXT_MULTI_LINE = "Multiple lines of text"
    INTEGER = "A whole number"
```

Just so they order a bit more nicely on the 'what kind of question' page 👀